packages/apollo-graphql/src/transforms.ts

```diff
@@ -94,13 +95,37 @@ function sorted<T>(
   return undefined;
 }
 
+// Predicate for filtering and type checking operation definitions (query, mutation, subscription)
+// Similar to graphql's isExecutableDefinitionNode
+function isOperationDefinitionNode(
+  node: ASTNode,
+): node is OperationDefinitionNode {
+  return node.kind === 'OperationDefinition';
+}
+
+// Predicate for filtering and type checking fragment definitions
+// Similar to graphql's isExecutableDefinitionNode
+function isFragmentDefinitionNode(
+  node: ASTNode,
+): node is FragmentDefinitionNode {
+  return node.kind === 'FragmentDefinition';
```

Comment:
FYI: You can import `types` from `graphql-js` and use `Types.FragmentDefinition` with type support

https://github.com/graphql/graphql-js/blob/0f3c1777100e9606f5e0e9b509e6e2a807b137ad/src/language/kinds.js#L19

Reply:
Awesome! I looked for the typed string literal and they aren't even using it where I looked.

Reply:
No longer need these predicates at all